docs/configuring-playbook-bridge-mautrix-wsproxy.md

```diff
@@ -0,0 +1,22 @@
+# Setting up Mautrix wsproxy (optional)
+
+The playbook can install and configure [mautrix-wsproxy](https://github.com/mautrix/wsproxy) for you.
+
+See the project's [documentation](https://github.com/mautrix/wsproxy#readme) to learn what it does and why it might be useful to you.
+
+Use the following playbook configuration:
+
+```yaml
+matrix_mautrix_wsproxy_enabled: true
+matrix_mautrix_androidsms_appservice_token: 'random string'
+matrix_mautrix_androidsms_homeserver_token: 'random string'
+matrix_mautrix_imessage_appservice_token: 'random string'
+matrix_mautrix_imessage_homeserver_token: 'random string'
```

Comment:
Why aren't these tokens auto-generated in `group_vars/matrix_servers` like all other bridge tokens?

See an example here: https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/75d4b534844fbcb0df03ad34a2496115786d1529/group_vars/matrix_servers#L501-L503

Reading below, it seems like you need to make these tokens match whatever the `mautrix-imessage` bridge on Mac/Android hardcodes? So these are not really random strings and this part should be updated.